synapse/tests/test_lib_ast.py

```diff
@@ -1473,3 +1473,7 @@ async def test_ast_subgraph_light_edges(self):
             self.len(1, nodes)
             self.len(1, nodes[0][1]['path']['edges'])
             self.eq('refs', nodes[0][1]['path']['edges'][0][1]['verb'])
+
+            msgs = await core.stormlist('media:news | graph --no-edges')
+            nodes = [m[1] for m in msgs if m[0] == 'node']
+            self.len(0, nodes[0][1]['path']['edges'])
```

Comment:
Is there only one node expected as output? because this feels like an order-dependent result.

Reply:
Yea, there's only 1 media:news node in the cortex